test/gurka/test_loops.cc

```diff
@@ -161,3 +162,43 @@ TEST(loops, please_god_why) {
   auto map = gurka::buildtiles(layout, ways, {}, {}, "test/data/gurka_please_god_why", build_config);
   check_opposing(map, 12);
 }
+
+TEST(loops, use_kCuldesac_false) {
+  const std::string ascii_map = R"(
```

Comment:
The map looks like https://user-images.githubusercontent.com/50918075/110827032-b9e66c00-82a6-11eb-9673-6f418163a19b.png